charactersheet/charactersheet/index.html

```diff
@@ -73,100 +73,131 @@ <h2 data-bind='text: playerTitle'></h2>
 				</div>
 			</div>
-			<img src="/images/heart.png" height="35" style="margin-top:-12px;" />&nbsp;
-			<h3 style="display:inline;">
-				Health &amp; Stats
-			</h3>
-      <hr />
-			<div class="row">
-				<div class="col-md-8">
-					<div id="stats" data-bind="template: { name: 'stats.tmpl', templateUrl: 'stats', data: statsViewModel }">
-					</div>
+			<div class="col-md-4 col-sm-12 col-padded">
+				<div id="spell_stats" data-bind="template: { name: 'spell_stats.tmpl', templateUrl: 'spell_stats', data: spellStatsViewModel }">
 				</div>
-				<div class="col-md-4">
-					<div id="spell_stats" data-bind="template: { name: 'spell_stats.tmpl', templateUrl: 'spell_stats', data: spellStatsViewModel }">
-					</div>
+			</div>
+			<div class="col-md-4 col-sm-12 col-padded">
+				<div id="saving_throws" data-bind="template: { name: 'saving_throws.tmpl', templateUrl: 'saving_throws', data: savingThrowsViewModel }">
 				</div>
 			</div>
-			<img src="/images/shield.png" height="35" style="margin-top:-12px;" />&nbsp;
-			<h3 style="display:inline;">
-				Equipment, Skills, &amp; Treasure
-			</h3>
-      <hr />
-			<div class="row">
-				<div class="col-md-4 col-sm-6 col-padded">
-					<div id="spell_slots" data-bind="template: { name: 'spell_slots.tmpl', templateUrl: 'spell_slots', data: spellSlotsViewModel }">
-					</div>
-					<div id="saving_throws" data-bind="template: { name: 'saving_throws.tmpl', templateUrl: 'saving_throws', data: savingThrowsViewModel }">
-					</div>
-					<div id="skills" data-bind="template: { name: 'skills.tmpl', templateUrl: 'skills', data: skillsViewModel }">
-					</div>
+		</div>
+	</div>
+<!-- End Stats Tab -->
+	<div role="tabpanel" class="tab-pane" id="skills" data-bind="with: skillsTabViewModel, css: skillsTabStatus">
+		<div class="row">
+			<div class="col-md-6 col-sm-12 col-padded">
+				<div data-bind="template: { name: 'feats_prof.tmpl', templateUrl: 'feats_prof', data: featsProfViewModel }">
 				</div>
-				<div class="col-md-8 col-sm-6 col-padded">
-					<div id="equipment" data-bind="template: { name: 'equipment.tmpl', templateUrl: 'equipment', data: equipmentViewModel }">
-					</div>
-					<div id="spells" data-bind="template: { name: 'spells.tmpl', templateUrl: 'spells', data: spellbookViewModel }">
-					</div>
-					<div data-bind="template: { name: 'treasure.tmpl', templateUrl: 'treasure', data: treasureViewModel }">
-					</div>
+			</div>
+			<div class="col-md-6 col-sm-12 col-padded">
+				<div id="skills" data-bind="template: { name: 'skills.tmpl', templateUrl: 'skills', data: skillsViewModel }">
 				</div>
 			</div>
-			<div class="row">
-				<div class="col-sm-6 col-padded">
-					<div id="weapons" data-bind="template: { name: 'weapons.tmpl', templateUrl: 'weapons', data: weaponsViewModel }">
-					</div>
+		</div>
+	</div>
+<!-- End Skills Tab -->
+	<div role="tabpanel" class="tab-pane" id="spells" data-bind="with: spellsTabViewModel, css: spellsTabStatus">
```

Comment:
It looks like spells and spell-slots are not in a row or something. THey're too close to the edges. That means that its either not in a column or row.
![screenshot 2016-02-03 09 58 26](https://cloud.githubusercontent.com/assets/3310280/12791803/8cd7a5d2-ca5d-11e5-9fb6-dc23cd17181b.png)